src/prerender.js

```diff
@@ -1,9 +1,9 @@
-const fs = require('fs');
-const { resolve } = require('path');
+// const fs = require('fs');
```

Comment:
Shouldn't we remove these lines? (nit)

Reply:
I left them in for now because they're hopefully going to get used to inline prerendered data now that we're on preact-cli 3 rc.2.